content/800-data-platform/05-data-browser.mdx

```diff
@@ -154,18 +164,17 @@ Use the **Filters** menu to filter data in the model by adding conditions.
       - **gt**
       - **gte**
       - **not**
-   3. Type the value you want to use for the filter.<br />
-      **Step result**: The model is filtered immediately after you define the first filter.
+   3. Type the value you want to use for the condition.<br />
+      **Step result**: The data in the model is updated immediately based on the condition.
```

Comment:
```suggestion
      **Step result**: The data in the model updates immediately based on the condition.
```

Reply:
Can we recast the sentence to avoid passive voice? So something like "the Data Browser updates the data in the model immediately, based on the condition."

Reply:
Thanks @andrew-walford-prisma and @tanberry. Changed as Andrew suggested.